.gitattributes

```diff
@@ -0,0 +1 @@
+* text eol=lf
```

Comment:
Can you make sure this does not affect .bat files? Just in case 🤞

Reply:
Can also remove from here and make a separate pull request to have proper discussion. Currently the issue is that without this, spotless will try to fix all LF settings to CRLF on Windows even if in git repository settings, `autocrlf` is not enabled.

Reply:
Removed for now.